pokemon/src/main/java/gamarket/EncounterGUI.java

```diff
@@ -1,125 +1,1075 @@
+        fightMenu.setPadding(new Insets(50, 0, 0, 185));
+        
+        // Add everything to the main GUI container: the pokemons, menu, and text box. 
+        gp.add(poke, 0, 0);
+        gp.add(menu, 1, 1);
+        gp.add(mySP, 0, 1);
+        gp.setStyle("-fx-alignment: center-right;");
+        
+        // Style them health bars and add them to the main GridPane GUI 'gp'.
+        StackPane playerPokeHealthBar = new StackPane();        // The player pokemon. 
+        playerPokeHealthBar.getChildren().add(hp2);
+        gp.add(playerPokeHealthBar, 1, 0);
+        playerPokeHealthBar.setPadding(new Insets(0, 0, 532, 300));
+        hp2.setStyle("-fx-background-radius: 300px;" + "-fx-pref-width: 300px; " + "-fx-border: none; "
+            + "-fx-background-color: white;" + "-fx-border-radius: 300px;" + "-fx-border-width: 7px;"
+            + "-fx-font-size: 24px;" + "-fx-text-align: center;" + "-fx-text-base-color: rgb(0,0,0);");
+ 
+        StackPane wildPokeHealthBar = new StackPane();          // The wild pokemon. 
+        wildPokeHealthBar.getChildren().add(hp1);
+        gp.add(wildPokeHealthBar, 0, 1);
+        wildPokeHealthBar.setPadding(new Insets(0, 1035, 160, 0));  
+        hp1.setStyle("-fx-background-radius: 300px;" + "-fx-pref-width: 300px; " + "-fx-border: none; "
+            + "-fx-background-color: white;" + "-fx-border-radius: 300px;" + "-fx-border-width: 7px;" 
+            + "-fx-font-size: 24px;" + "-fx-text-align: center;" + "-fx-text-base-color: rgb(0,0,0);");
+ 
+        // Creating the pokemon party 
+        pokemonMenu = new VBox(7); 
+        pokemonMenu.setPadding(new Insets(0, 0, 0, 305));
+        this.createPokemonPartyUI();
+        this.styleThemPokemonButtons();
+        pokemonMenu.setVisible(false);
+        menu.getChildren().addAll(battleMenu, fightMenu, pokemonMenu);
+        
+        return gp;
+    }
+ 
+    /**
+     * Updates the Player's pokemon health, name, and level. 
+     */
+    public String updateHP1()
+    {
+        // update the player's pokemon status bars: name, level, health.
+        return playerPokemon.getIdentStats().getName() +
+                "       Lv." + Integer.toString(playerPokemon.getIdentStats().getLevel()) 
+                + "\n    " + Integer.toString(playerPokemon.getDefensiveStats().getHPCurrent()) + " / "
+                + Integer.toString(playerPokemon.getDefensiveStats().getHP()) + " HP";
+    }
+ 
+    /**
+     * Updates the Wild's pokemon health, name, and level. 
+     */
+    public String updateHP2()
+    {
+        // update the wild's pokemon status bars: name, level, health. 
+        return wildPokemon.getIdentStats().getName() +
+                "       Lv." + Integer.toString(wildPokemon.getIdentStats().getLevel()) 
+                + "\n    " + Integer.toString(wildPokemon.getDefensiveStats().getHPCurrent()) + " / "
+                + Integer.toString(wildPokemon.getDefensiveStats().getHP()) + " HP";
+    }

```

Comment:
these two functions are basically the same except the pokemon used. make updateHP(Pokemon poke) to make code drier